eval/README.md

```diff
@@ -0,0 +1,32 @@
+## Datasource
+
+Dataset was acquired from [here](https://research.google/tools/datasets/coached-conversational-preference-elicitation/). It consists of 12,000 annotated utterances over 502 conversations. 
+
+## Results
+
+This are the results over all conversations in the data set. Reported values are for micro-averaged precision, recall and f1 scores.
```

Comment:
Would be nice to quote the specific command (i.e., parameterization of the evaluation script) that was used for generating these results and just include the output of that script in a verbatim block